src/map/options.ts

```diff
@@ -20,11 +20,11 @@ import { COLOR_MAPS } from './colorscales';
 import BARS_SVG from '../static/bars.svg';
 import HTML_OPTIONS from './options.html.in';
 
-// in 3D mode, only strings are supported for 'marker.symbol', and only very few
-// of them. See https://github.com/plotly/plotly.js/issues/4205 as the plotly
-// issue tracking more symbols in 3D mode.
-const POSSIBLE_SYMBOLS_IN_3D = ['circle', 'square', 'diamond', 'cross', 'x'];
-
+// in 3D mode, only strings are supported for 'marker.symbol'.
+/* eslint-disable */
+const markers3d = require('./plotly/markers3d');
```

Comment:
Why not `import` here?

Reply:
can't get it to work as an import from both javascript and typescript in any other way. if you know how to do it, change it, I couldn't figure anything else that worked